slurmToRuby.py

```diff
@@ -0,0 +1,19 @@
+import re
+fName = "squeueDataRaw.csv"
+numCore = 0
+userCores = {}
+with open(fName) as f:
+     lines = f.readlines()
+for i in lines:
+
+    numCore = (int(re.sub(' +', ' ', i).split(' ')[1]))
+    userName = re.sub(' +', ' ', i).split(' ')[0]
+    if userName in userCores:    
+        userCores[userName] = numCore + userCores.get(userName)
```

Comment:
I think you can just do
```suggestion
        userCores[userName] += numCore
```